llm-request-plz.el

```diff
@@ -113,7 +113,7 @@ the curl process and an error message."
                     (cdr curl-error))))
         (t (user-error "Unexpected error: %s" error))))
 
-(cl-defun llm-request-plz-async (url &key headers data on-success on-success-raw media-type
```

Comment:
Since we talked yesterday about using media types consistently, I removed this. There shouldn't be any need to parse the response manually.